internal/common/labels.go

```diff
@@ -22,11 +22,12 @@ func (a AppComponent) String() string {
 }
 
 const (
-	AppComponentMonitoring      AppComponent = "monitoring"
-	AppComponentSchedule        AppComponent = "schedule"
-	AppComponentTemplating      AppComponent = "templating"
-	AppComponentTektonPipelines AppComponent = "tektonPipelines"
-	AppComponentTektonTasks     AppComponent = "tektonTasks"
+	AppComponentMonitoring          AppComponent = "monitoring"
+	AppComponentSchedule            AppComponent = "schedule"
+	AppComponentTemplating          AppComponent = "templating"
+	AppComponentTektonPipelines     AppComponent = "tektonPipelines"
+	AppComponentTektonTasks         AppComponent = "tektonTasks"
+	AppComponetVMDeletionProtection              = "VMDeletionProtection"
```

Comment:
Please keep this consistent, and fix the typo:
```
AppComponentVMDeletionProtection AppComponent = "vmDeletionProtection"
```

Reply:
Done.